### PR TITLE
First rest-calls working

### PR DESCRIPTION
**Pull request by anssi55**
This branch was all about getting first two rest-api calls working.
Couple typeorm entities were added to able to get the calls working.

### CODE REVIEW

File: app/app.ts [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
+    port: 3306,
+    username: "newuser",
+    password: "password",
+    database: "bulletinboard2",
+    entities: [
+        Posts, Users, Categories
+    ],
+    synchronize: true,
+    logging: false}).then(connection => {
+    connection.getRepository(Posts);
```
Comment (henrikra) [inline app/app.ts]: Is this line doing something? 🤔

File: app/app.ts [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
+    app.use(bodyParser.urlencoded({ extended: true }));
+    app.use(bodyParser.json());
 
-app.use('', Index);
+    app.use('/', Index);
 
-app.listen(3000);
+    app.listen(3000);
+}).catch(error => console.log(error));
 console.log("Server running on: http://localhost:" + 3000 + "/");
```
Comment (henrikra) [inline app/app.ts]: Hmm this log runs always even though server could not be started. Could you move it to proper place? :D

**Review by henrikra — COMMENTED** [review]
Couple things 👍

File: ormconfig.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+   "type": "mysql",
+   "host": "localhost",
+   "port": 3306,
+   "username": "newuser",
```
Comment (henrikra) [inline ormconfig.json:5]: So is the TypeORM now using this configuration or the one in the code? 🤔

File: app/app.ts (hunk 1 above)
Comment (anssi55) [reply app/app.ts]: All these are now fixed ☼

**Review by henrikra — APPROVED** [review]
Looks good to me

**Pull request merged**